packages/manifest-schemas/formats/web3api.plugin/0.0.1-prealpha.1.json

```diff
@@ -9,27 +9,33 @@
   ],
   "properties": {
     "format": {
+      "description": "Polywrap plugin YAML format version.",
       "type": "string",
       "const": "0.0.1-prealpha.1"
     },
     "language": {
+      "description" : "Plugin language.",
       "type": "string",
       "format": "pluginLanguage"
     },
     "schema": {
+      "description": "Path to graphql schema.",
       "type": "string",
       "format": "file"
     },
     "import_redirects": {
+      "description": "Redirects enabling the import of plugins.",
       "type": "array",
       "items": {
         "type": "object",
         "additionalProperties": false,
         "properties": {
           "uri": {
+            "description": "URI resolving to the imported plugin schema.",
             "type": "string"
           },
           "schema": {
+            "description": "Graphql schema for imported plugin.",
```

Comment:
Path to GraphQL schema of the module to which URI will be redirected

Reply:
Updated.